src/main/java/net/glowstone/block/entity/state/GlowBeacon.java

```diff
@@ -28,7 +30,7 @@ private BeaconEntity getBlockEntity() {
 
     @Override
     public Collection<LivingEntity> getEntitiesInRange() {
-        return getWorld().getNearbyEntitiesByType(Player.class, getLocation(), getTier() * 10 + 10, getWorld().getMaxHeight());
+        return getWorld().getNearbyEntitiesByType(Player.class, getLocation(), getTier() * RADIUS_MULTIPLIER + 10, getWorld().getMaxHeight());
```

Comment:
Assuming the formula is equivalent to `(tier + 1) * RADIUS_MULTIPLIER`, it should be 
```suggestion
        return getWorld().getNearbyEntitiesByType(Player.class, getLocation(), getTier() * RADIUS_MULTIPLIER + RADIUS_MULTIPLIER, getWorld().getMaxHeight());
```